### PR TITLE
WIP DM-5581 Document v1

### PR DESCRIPTION
**Pull request by jonathansick**
See http://sqr-011.lsst.io/en/tickets-dm-5581/

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -2,6 +2,613 @@
...
+
+How are tools selected
+========================
+
+With all communication tools, there is always a balance of providing
+users a diversity of tools that they want to use and consolidating
+tools so that information does not fragment and to boost the network
+effect. We have found that the parameter space of task, synchronicty
+and audience do require multiple tools, but we favour only having one
+supported tool for each locus in that space.
```
Comment (ktlim) [inline index.rst]: Would be nice to have a chart showing combinations of task, synchronicity, and audience and which tool satisfies each combination (possibly an extension of the chart below).

File: index.rst [hunk 2]
```diff
@@ -2,6 +2,613 @@
...
+DM uses Confluence wikis, although their role is being diminished with
+the introduction of |clo| and the |ltd| publishing paradigm (including
+Technical Notes, the new Developer Guide and software documentation).
+
+SQuaRE dissuades software documentation in wikis, since it cannot be
+managed with standard software release tools, cannot be tested by our
+continuous integration harness, is "out of sight out of mind" for the
+developers, and is hard to maintain. We are in the process of
+migrating all software documentation from Confluence to other, better
+harnesses.
```
Comment (ktlim) [inline index.rst]: These conditions don't seem to necessarily apply to more design-level documentation, where it's not obvious that the GitHub platform is better than tools that provide inline comment threads like Confluence and Google Docs.  (P.S.  Google Docs seems to be used a lot informally; should it be mentioned here somewhere?)

Comment (jonathansick) [inline index.rst]: What is Google Docs being used for?

File: index.rst (hunk 1 above)
Comment (jonathansick) [inline index.rst]: Good idea. I'll make slides first, and see if such a chart can emerge from that.

File: index.rst (hunk 2 above)
Comment (ktlim) [inline index.rst]: I've seen it being used for 1) planning spreadsheets, 2) design documents (both within DM and LSST-wide in Operations planning and systems engineering).
Some people are also using Dropbox to pass around information like presentations.

Comment (timj) [inline index.rst]: I am assuming that Dropbox for presentations and google docs for design documents are not where the final thing ends up though? I think this discussion is where the results of the writing become permanent.

Comment (ktlim) [inline index.rst]: They're still communications mechanisms in use.

I don't think that it's yet clear that documents are generally migrating out of Google Docs.  I think I recall some that have turned into DocuShare Word docs.

Comment (timj) [inline index.rst]: I'd be pretty upset if google docs became the final thing that is linked to. Exporting to docushare is fine.

Comment (jbecla) [inline index.rst]: I think google docs have a place in our project, and they fill an important niche. We badly need place where we could keep things like presentations and all these little drawings and diagrams, and docushare is just not good for it. Yes we are putting the official docs in docushare (baseline docs, fdr presentations etc) but when it comes to less official docs, like all these qserv presentations, all these little diagrams that we used for baseline, they never made it there. One big issue with docushare is that is impossible to find things there, you have to retrieve to make any change etc. It is also so super convenient to share a doc, update it live, from mac, windows and linux. And that is just not there in docushare.

**Comment by timj** [comment]
I have a comment in my inbox that @jbecla made on this PR on March 22 but it seems to have disappeared from the PR history. I really don't think that things should end up on Google Doc as the end location. They aren't discoverable there at all. How do we index them? What are these "Qserv presentations" that are being mentioned? We should make presentations more visible. One of the problems we have is that people treat presentations as transient things that just slowly disappear. I don't see how Google Docs is more discoverable than docushare. I absolutely agree that Google Docs is great for live sharing and updating in a cross platform manner. We need to solve the end game though. I'm putting all DM presentations that are presented externally on Zenodo. Maybe we need to talk about the persisting of internal presentations.

**Comment by jonathansick** [comment]
Do we have any access-control concerns on our internal presentations? If not, then we can also put them on Zenodo. The goal of [DocHub](http://sqr-011.lsst.io/en/latest/#a-documentation-index) is to provide a unified LSST/DM API and front-end to everything in Zenodo, and elsewhere.

btw, should we have this conversation on https://community.lsst.org/t/technote-sqr-011-dm-communication-platforms/616?u=jsick since this is a closed PR?

**Comment by ktlim** [comment]
@timj Jacek's comment shows up for me under "ktlim and 3 others commented on an outdated diff".

**Pull request merged**